### PR TITLE
Note that Safari has fixed flexbug 1.

### PR DESCRIPTION
**Pull request by grorg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,11 +40,10 @@ As the spec continues to evolve and vendors nail down their implementations, thi
     <td>
       Chrome (fixed in 44)<br>
       Opera (fixed in 31)<br>
-      Safari
+      Safari (fixed in 10)
     </td>
     <td>
-      <a href="https://code.google.com/p/chromium/issues/detail?id=426898">Chrome #426898</a><br>
-      <a href="https://bugs.webkit.org/show_bug.cgi?id=146020">Safari #146020</a>
```
Comment (philipwalton) [inline README.md]: Please don't remove the link to the bug. I think it's good to have it there for historical context.

**Comment by philipwalton** [comment]
Thanks, I may hold off on merging this until Safari 10 is actually released.

**Comment by gregwhitworth** [comment]
@grorg Hey, can you add back in the link as @philipwalton suggested and we'll merge it in. Thanks.

**Comment by philipwalton** [comment]
Fixed via 6a2222982bbb8f22fc0c90577be0f42b63dc37cd.

**Comment by grorg** [comment]
Hey, sorry I didn't do this for you. I've been out for a few days. Thanks for taking it!

**Pull request closed without merging**